methods/ev.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "ev",
+  "status": "registered",
+  "specification": "https://github.com/KayTrust/did-method-ev",
+  "contactName": "David Ammouial (NTT DATA)",
+  "contactEmail": "<firstName>.<lastName>@nttdata.com",
```

```suggestion
  "contactEmail": "<firstName>.<lastName>@nttdata.com",
```

Please include a specific contact email address.